docs/sharding.md

```diff
@@ -101,7 +101,98 @@ namespace SimpleChatV3
 }
 ```
 
-For other languages such as JavaScript, we will support route algorithm customization in the future.
+> The dependency of customized negotiation router: 
+> 
+> If you need to implement [`GetNegotiateEndpoint(HttpContext context, IEnumerable<ServiceEndpoint> endpoints)`](https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR/EndpointRouters/IEndpointRouter.cs) method yourself and rely on the parameter `HttpContext`, for example, use `HttpContext.Request.Query["endpoint"]` to select a nearer endpoint for router, you can only use the HTTP trigger in your negotiation functions, so that your router can get the `HttpContext` object correctly.
+
+#### Other languages
+
+For languages other than C#, we support specifying target endpoints in each request. You will use new binding types to get endpoint information.
```

Comment:
a new binding type called `SignalRNegotiation`?

Reply:
to get multi-endpoints information?

Reply:
There are two new bindings, one is `SignalRNegotiation`, another is `SignalREndpoints`. I'd like to introduce the detail information in the following content.